dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -362,6 +379,25 @@ class BGPPeerMgr(Manager):
         os.remove(tmp_filename)
         return rc == 0
 
+    def get_local_interface(self, local_addr):
+        local_addresses = self.directory.get_slot("local_addresses")
+        # Check if the local address of this bgp session has been set
+        if local_addr not in local_addresses:
+            return None
+        local_address = local_addresses[local_addr]
+        interfaces = self.directory.get_slot("interfaces")
+        # Check if the information for the interface of this local address has been set
+        if local_address.has_key("interface") and local_address["interface"] in interfaces:
+            return interfaces[local_address["interface"]]
+        else:
+            return None
+
+    def get_vnet(self, interface):
```

Comment:
this method does not really belong to here. it belongs to InterfaceMgr

Reply:
Done it